tests.toml

```
@@ -0,0 +1,26 @@
+test_format = 1.0
+
+[default]
+
+    # ------------
+    # Tests to run
+    # ------------
+
+    exclude = ["install.multi", "install.subdir", "install.nourl", "change_url"]
```

```suggestion
    exclude = ["install.multi", "change_url"]
```

Shouldnt be needed to list them in exclude, they are already not relevant for this app